### DISCUSSION TITLE
Please update translations more frequently

### DISCUSSION THREAD

**Discussion by raspin0** [linuxdeepin/developer-center#4448] [General | 常规讨论]
Hello, could you please work on automating the process of upstreaming translations from transifex? Currently it takes way too long for localisation to be upstreamed to the final release. I believe that the translations of entire project should be updated monthly or every couple of months. Currently it's very difficult to keep up motivation for volunteering translations if you have to wait years until it's get updated. 

Edit: Also as I'm opening up this issue, I'd want to mention that there are a lot of missing translations in the App Store. We don't need to translate all of the descriptions, but allow to translate entire UI of the application.

**Comment by BLumia** [comment]
cc @myml for the App Store translation issue.

About the translation in general, currently we are doing translations manually by project maintainers. We do plan to check if we can pull the translations automatedly. But before that happen, if you find some project lacking translations, you can tell us via Matrix/Telegram chat or create an issue, and I'll help you forward the information to crossponding team :)
